### ISSUE TITLE
Connection Module for FMS

### ISSUE THREAD

**Issue by JonasPetersenFAU** [amosproj/amos2022ss03-turtlebot-fleet-management#52]
## User story
1. As a SD
2. I want / need  to develop a connection module for FMS
3. So that it establishes MQTT connection with broker using mqtt_broker and de/encode msgs using vda5050_msgs
## Acceptance criteria
* Connection module is developed

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
